frontend/src/views/RapportPage.vue

```diff
@@ -952,9 +952,9 @@ export default defineComponent({
       />
     </div>
     <div class="fr-col-lg-4 fr-col-xl-4">
-      <ImagePresentation v-if="isRapportVendeur" :src="images.rapportVendeurSvg" alt="Illustration de la page du rapport vendeur HistoVec" />
+      <ImagePresentation v-if="isRapportVendeur" :src="images.rapportVendeurSvg" alt="" />
```

Comment:
On peut complétement supprimer l'attribut. Dans le composant ImagePresentation le alt n'est pas obligatoire et a une valeur par défaut à ''.

```suggestion
      <ImagePresentation v-if="isRapportVendeur" :src="images.rapportVendeurSvg" />
```